### PR TITLE
[1-17] UI: Set up and config  Angular Material, Less, Linter

### PR DESCRIPTION
**Pull request by HrebenYury**

### CODE REVIEW

File: exalearn-ui/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "printWidth": 120,
+    "singleQuote": true,
+    "endOfLine": "auto",
+    "trailingComma": "none",
+    "arrowParens": "avoid",
```
Comment (YanaBr) [inline exalearn-ui/.prettierrc]: Please, change "arrowParens": "always", it's a good practice

File: exalearn-ui/.prettierrc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "printWidth": 120,
+    "singleQuote": true,
+    "endOfLine": "auto",
+    "trailingComma": "none",
```
Comment (YanaBr) [inline exalearn-ui/.prettierrc:5]: Please, add "jsxBracketSameLine": true, "useTabs": true, "tabWidth": 4, "semi": true options

File: exalearn-ui/src/styles.less [hunk 3]
```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
```
Comment (YanaBr) [inline exalearn-ui/src/styles.less]: Please, refactor file and make styles definitions multiline:
```
html, body {
    height: 100%;
}
body { 
     margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; 
}
```

**Review by YanaBr — CHANGES_REQUESTED** [review]

**Review by KotdenGit — DISMISSED** [review]

**Review by GeranSkot — DISMISSED** [review]
noice

File: exalearn-ui/.prettierrc (hunk 2 above)
Comment (HrebenYury) [reply exalearn-ui/.prettierrc:5]: ok, i got it

File: exalearn-ui/src/styles.less (hunk 3 above)
Comment (HrebenYury) [reply exalearn-ui/src/styles.less]: my bad

**Review by YanaBr — APPROVED** [review]

**Review by KuznetsovR — APPROVED** [review]

**Review by GeranSkot — APPROVED** [review]

**Review by TanyaKulikovskaya — APPROVED** [review]

**Review by GeranSkot — APPROVED** [review]

**Review by GeranSkot — APPROVED** [review]

**Review by GeranSkot — APPROVED** [review]

**Pull request merged**